rhobs/alerting/data_plane/prometheus.latency_release_creation_alerts.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-latency-release-creation-alerting-rules
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: latency-release-creation
+    interval: 1m
+    rules:
+    - alert: LatencyReleaseCreation
+      expr: |
+        (
+          sum by (source_cluster) (rate(integration_svc_release_latency_seconds_bucket{le="+Inf"}[5m]))
+          -
+          sum by (source_cluster) (rate(integration_svc_release_latency_seconds_bucket{le="10"}[5m]))
+        ) /
+         sum by (source_cluster) (rate(integration_svc_release_latency_seconds_bucket{le="+Inf"}[5m]))
```

Comment:
Are we expecting more than one time series per cluster for those queries (`integration_svc_release_latency_seconds_bucket{le="X"}`)?
Otherwise, no need to use `sum`.

Reply:
the info on the legend is much easier to read with the sum by (source_cluster). So for clarity and presentation for the reader I put it in. 

![image](https://github.com/redhat-appstudio/o11y/assets/113446385/723b3cbb-7896-4d30-a9d3-b2a397f4a377)

this is what it looks like without 

![image](https://github.com/redhat-appstudio/o11y/assets/113446385/6b7cd30d-5db6-4399-8ee8-a4b91ce69ccf)

If the presentation doesn't matter so much thats no problem, I'll just remove it. It doesn't change the relevant information, just how it looks on the panel.

Reply:
There's no presentation in this context, so it indeed doesn't matter.

Reply:
sum removed. Thanks for your guidance.